scripts/Interface/ui/drill/model/specifications.py

```diff
@@ -0,0 +1,210 @@
+    FIGARO = "FIGARO"
+
+    # techniques
+    SANS = "SANS"
+    REFL = "Reflectometry"
+
+    # acquisition modes
+    SANS_ACQ =   "SANS"
+    REFL_POL =   "Polarized"
+    REFL_NPOL =  "Unpolarized"
+
+    # correspondance between instrument and technique
+    TECHNIQUE = {
+            D11:    SANS,
+            D16:    SANS,
+            D22:    SANS,
+            D33:    SANS,
+            D17:    REFL,
+            FIGARO: REFL
+            }
+
+    # correspondance between instrument and acquisition mode
+    ACQUISITION_MODES = {
+            D11:    [SANS_ACQ],
+            D16:    [SANS_ACQ],
+            D22:    [SANS_ACQ],
+            D33:    [SANS_ACQ],
+            D17:    [REFL_POL, REFL_NPOL],
+            FIGARO: [REFL_NPOL]
+            }
+
+    # parameters for each acquisition mode
+    COLUMNS = {
+            SANS_ACQ: [
+                "SampleRuns",
+                "SampleTransmissionRuns",
+                "AbsorberRuns",
+                "BeamRuns",
+                "FluxRuns",
+                "ContainerRuns",
+                "ContainerTransmissionRuns",
+                "TransmissionAbsorberRuns",
+                "TransmissionBeamRuns",
+                "MaskFiles",
+                "ReferenceFiles",
+                "OutputWorkspace",
+                "WedgeWorkspace",
+                "SampleThickness",
+                "CustomOptions"
+                ],
+            REFL_POL: [
+                "Run00",
+                "Run01",
+                "Run10",
+                "Run11",
+                "DirectRun",
+                "OutputWorkspace",
+                "AngleOption",
+                "Theta",
+                "SummationType",
+                "CustomOptions",
+                "GlobalScaleFactor",
+                "UseManualScaleFactors",
+                "ManualScaleFactors"
+                ],
+            REFL_NPOL: [
+                "Run",
+                "DirectRun",
+                "OutputWorkspace",
+                "GlobalScaleFactor",
+                "UseManualScaleFactors",
+                "ManualScaleFactors",
+                "AngleOption",
+                "Theta",
+                "SummationType",
+                "CustomOptions"
+                ]
+            }
+
+    # algo name for each acquisition mode
+    ALGORITHM = {
+            SANS_ACQ:   "SANSILLAutoProcess",
+            REFL_POL:   "ReflectometryILLAutoProcess",
+            REFL_NPOL:  "ReflectometryILLAutoProcess"
+            }
+
+    # ideal number of threads for each acquisition mode (optional).
+    # If not provided, Qt will decide
+    THREADS_NUMBER = {
+            }
+
+    # settings for each acquisition mode
+    SETTINGS = {
+            SANS_ACQ : {
+                "ThetaDependent": True,
+                "SensitivityMaps": "",
+                "DefaultMaskFile": "",
+                "NormaliseBy": "Timer",
+                "SampleThickness": "0.1",
+                "BeamRadius": "0.05",
```

Comment:
I think this was too small value for the default, make it 0.1 and also in the algorithm. You can `BeamRadius=0.05` to the existing tests, so that they still pass as they are.

Reply:
I change the default value to 0.1.
0.05 was taken from https://docs.mantidproject.org/nightly/algorithms/SANSILLAutoProcess-v1.html. We should also change that table (?)

Reply:
It's enough to change the default value in the algorithm. (And adjusts its tests as I mentioned above).